### ISSUE TITLE
Tabs issues

### ISSUE THREAD

**Issue by hotgeart** [SimpleBasicElements/Elements#2]
# Keyboard

When I press [Home] or [End] this should not move the scrollbar.

# Screen reader

This is wrong:

    #buttons [role='tabpanel'][hidden='hidden'] {
        display: block;
        //....
    }

`[hidden='hidden']` should be `display: none;` otherwise the screen reader will read the content.

If you want to test by yourself you can download [NVDA](https://www.nvda-fr.org/cat.php?id=2). You can make it read the text with arrow keys.

**Comment by Grafikart** [comment]
I added this display to handle animation. Would adding aria-hidden would prevent NVDA from reading this content ?
I can add this attribute so if people use strange CSS rules we can still have an accessible system.

**Comment by hotgeart** [comment]
Yes `aria-hidden="true"` will do the job

**Comment by Grafikart** [comment]
I just checked and you were absolutely right hidden is ignored for display block. I'll add this to the test suite. From now on, an element is considered "not visible" if it has both hidden="hidden" and aria-hidden="true"
